modules/nf-core/megahit/main.nf

```diff
@@ -25,15 +25,6 @@ process MEGAHIT {
     def args = task.ext.args ?: ''
     def args2 = task.ext.args2 ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-
-    def restart = ""
-    if (task.attempt > 1) {
-        // Set of extra flags to restart the assembly process
-        restart = "--continue"
```

Comment:
Will this be covered in prefect, if this is removed from here (and metaspades)?

Reply:
Nextflow doesn't resure the folder on retries, so we can't ise use that the spades --continue flag. We will have to workout a solution, otherwise let nextflow restart the process from scratch.